README.md

```diff
@@ -115,7 +115,14 @@ make install prefix=/usr/local
 Prerequisites for compilation are:
 
 * `make`
-* [Go 1.9+](http://golang.org/doc/install)
+* [Go 1.11+](http://golang.org/doc/install)
+
+Depending on your environment, you may encounter a `build flag -mod=vendor only valid when using
```

Comment:
Instead of providing build instructions like “sometimes you need this; sometimes you need that”, how about we simply change our build step (i.e. `make`) to work no matter the environment?

Ref. https://github.com/github/hub/pull/2142

Reply:
👍 makes sense to me